docs/user/security/authentication/index.asciidoc

```diff
@@ -211,3 +212,27 @@ leaked, it can't be re-used after logout. This is known as "local" logout.
 {kib} can also initiate a "global" logout or _Single Logout_ if it's supported by the external authentication provider and not
 explicitly disabled by {es}. In this case, the user is redirected to the external authentication provider for log out of
 all applications associated with the active provider session.
+
+[[kerberos]]
+==== Kerberos Single Sign-On
+
+As with the previous SSOs, make sure that you have configured {es} first accordingly. For the Kerberos authentication realm, please see {xpack-ref}/kerberos-realm.html[Configuring a Kerberos realm]
```

Comment:
This page has moved to https://www.elastic.co/guide/en/elasticsearch/reference/master/kerberos-realm.html, so I've updated the URL here:
```suggestion
As with the previous SSOs, make sure that you have configured {es} first accordingly. See {ref}/kerberos-realm.html[Kerberos authentication].
```